x-pack/plugins/actions/server/lib/related_saved_objects.ts

```diff
@@ -9,17 +9,17 @@ import { schema, TypeOf } from '@kbn/config-schema';
 import { Logger } from '@kbn/core/server';
 
 export type RelatedSavedObjects = TypeOf<typeof RelatedSavedObjectsSchema>;
+export type RelatedSavedObject = TypeOf<typeof RelatedSavedObjectSchema>;
 
-const RelatedSavedObjectsSchema = schema.arrayOf(
-  schema.object({
-    namespace: schema.maybe(schema.string({ minLength: 1 })),
-    id: schema.string({ minLength: 1 }),
-    type: schema.string({ minLength: 1 }),
-    // optional; for SO types like action/alert that have type id's
-    typeId: schema.maybe(schema.string({ minLength: 1 })),
-  }),
-  { defaultValue: [] }
-);
+const RelatedSavedObjectSchema = schema.object({
+  space_ids: schema.arrayOf(schema.string({ minLength: 1 })),
```

Comment:
Perhaps we should use `camelCase` at this level.

Reply:
Makes sense. Updated in https://github.com/elastic/kibana/pull/144111/commits/6071b10e13fc7ac979dff1eca6b57b211f0f7eba